docs/_Research-Data-Management/03-md.md

```diff
@@ -4,14 +4,51 @@ category: Research-Data-Management
 layout: default
 docs_css: markdown
 redirect_from: /Research-Data-Management
-empty: true
-hide: true
+empty: false
+hide: false
 ---
 
 # Metadata
 
+## Metadata is data about data
+
+Before we delve into specifications on what Metadata standards for the Microbiology community are or would be, let us explain what metadata is. 
+
+In general, metadata provides us with information about other data but does not tell us anything about the content of the data itself. Instead, it describes other types of information to help you understand or use the data you are working with. In the simplest terms, metadata is **data** about data.
+
+If you find our explanation lacking, we refer you to the [FAIR Cook Book](https://faircookbook.elixir-europe.org/content/home.html), more specifically to the [FAIR and the notion of metadata](https://faircookbook.elixir-europe.org/content/recipes/introduction/metadata-fair.html) section.
```

Comment:
```suggestion
For more details on the distinction between different types of metadata,  we refer you to the FAIR Cookbook recipe [FAIR and the notion of metadata](https://w3id.org/faircookbook/FCB068) section.
```

- removed the formulation that shoot ourserveles in the foot
- use the persistent identifier for the recipe